tests/test_ode.py

```diff
@@ -129,6 +129,48 @@ def test_initial_parameters(self):
         self.assertEqual([a0, c0], ode_model.initial_params)
         self.assertEqual([a0, k, l, m, p], ode_model.model_params)
 
+        print(ode_model.connectivity_mapping)
+        self.assertEqual({d: {l, c, m, d, a, a0},
+                          c: {k, a, p, c, l, m, d, c0, a0},
+                          a: {k, a, p, c, a0, d}},
+                         ode_model.connectivity_mapping)
+
+    def test_ODE_constraints(self):
+        """
+        Identical to test_polgar, but with a0 as free Parameter.
+        """
+        a, b, c, d, t = variables('a, b, c, d, t')
+        k, p, l, m = parameters('k, p, l, m')
+
+        a0 = Parameter('a0', min=0, value=10, fixed=False)
+        c0 = Parameter('c0', min=0, value=0.1)
+        b = a0 - d + a
+        model_dict = {
+            D(d, t): l * c * b - m * d,
+            D(c, t): k * a * b - p * c - l * c * b + m * d,
+            D(a, t): - k * a * b + p * c,
+        }
+
+        ode_model = ODEModel(model_dict, initial={t: 0.0, a: a0, c: c0, d: 0.0})
+
+        # Generate some data
+        tdata = np.linspace(0, 3, 1000)
+        # Eval
+        AA, AAB, BAAB = ode_model(t=tdata, k=0.1, l=0.2, m=.3, p=0.3, a0=10, c0=0)
+
+        constraints = [Ge(a0, c0)]
+
+        fit = Fit(ode_model, t=tdata, a=AA, c=AAB, d=BAAB)
```

Comment:
The constraints are not passed to `Fit`. It is probably better to use an equatilty or make the constraint the other way around, since initially the constraint is already true so we don't know if it was enforced properly.